modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/workflow-definitions/underwriting/workflow-definition.xml

```diff
@@ -0,0 +1,324 @@
+<?xml version="1.0"?>
+
+<workflow-definition
+	xmlns="urn:liferay.com:liferay-workflow_7.4.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="urn:liferay.com:liferay-workflow_7.4.0 http://www.liferay.com/dtd/liferay-workflow-definition_7_4_0.xsd"
+>
+	<version>10</version>
+	<condition>
+		<name>9247483e-463b-4f58-8482-f524cf1a50e8</name>
+		<description>Execute conditional logic.</description>
+		<metadata>
+			<![CDATA[
+				{
+					"xy": [
+						282.8981758043758,
+						283.4780023728212
+					]
+				}
+			]]>
+		</metadata>
+		<labels>
+			<label language-id="en_US">
+				isUnderwriter
+			</label>
+		</labels>
+		<script>
+			<![CDATA[import com.liferay.object.model.ObjectEntry;
+import com.liferay.object.service.ObjectEntryLocalServiceUtil;
+import com.liferay.portal.kernel.util.GetterUtil;
+import com.liferay.portal.kernel.workflow.WorkflowConstants;
+
+import java.io.Serializable;
+import java.util.Map;
+
+long classPK = GetterUtil.getLong((String)workflowContext.get(
+WorkflowConstants.CONTEXT_ENTRY_CLASS_PK));
+
+ObjectEntry objectEntry = ObjectEntryLocalServiceUtil.getObjectEntry(classPK);
+
+Map<String, Serializable> values = objectEntry.getValues();
+
+Serializable applicationStatus = values.get("applicationStatus");
```

Comment:
You just use once, you can put this as inline

Reply:
Done